doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
I would keep it until the next release.

On Thu, Jun 25, 2015 at 1:54 PM, abergeron <notifications@github.com> wrote:

> In doc/tutorial/extending_theano.txt
> <https://github.com/Theano/Theano/pull/3061#discussion_r33284167>:
>
> > @@ -259,7 +259,7 @@ Op Example
> >
> >          def make_node(self, x):
> >              # check that the theano version has support for __props__
> > -            assert hasattr(self, '_props')
> > +            assert hasattr(self, '__props__')
>
> My vote is to remove the check completely since __props__ is old enough
> that not many people should have a theano without it and it was in the
> latest release.
>
> But if we are to keep the check, then it should go back to _props, of
> course.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/3061/files#r33284167>.
>